.travis.rosinstall

```diff
@@ -7,6 +7,4 @@
 - git:
     uri: https://github.com/start-jsk/rtmros_common
     local-name: rtm-ros-robotics/rtmros_common
-- git:
-    uri: https://github.com/start-jsk/rtmros_tutorials
-    local-name: rtm-ros-robotics/rtmros_tutorials
+# rtmros_tutorials depends on rtmros_gazebo. So rtmros_tutorials is not included here.
```

Comment:
`rtmros_tutorials` is added at https://github.com/start-jsk/rtmros_gazebo/commit/2a8b8a4dcbf81323a3a0221250cf76065817e23d.
And I think `rtmros_tutorials` should be here, because when we want to change base packages in `rtmros_gazebo` (e.g., `hrpsys_gazebo_general`), we have to check this with `rtmros_tutorials`, which is what we use.
`rtmros_common` has tests from this thought: https://github.com/start-jsk/rtmros_common/blob/master/.travis.yml#L31

Reply:
Oops, I don't know that rtmros_gazebo/hrpsys_gazebo_atlas depends on rtmros_tutorials/hrpsys_ros_bridge_tutorials...
My previous comment is still effective, but I also think this dependency should be resolved.

Reply:
I think when we want to change base packages in `rtmros_gazebo` (e.g., `hrpsys_gazebo_general`), we **only** have to check this with `SampleRobot`, which can be simulated without `rtmros_tutorials` ???

Reply:
I see. I forgot about SampleRobot.
And I also forgot that HRP2 is closed robot and cannot be tested in this repository.

So problem of removing rtmros_tutorials seems mainly about hrpsys_gazebo_atlas.
You are preventing hrpsys_gazebo_atlas from being built, aren't you?
I'm against this idea, because we will hardly notice when we break
hrpsys_gazebo_atlas until we explicitly check travis of rtmros_tutorials.
I think this makes travis meaningless.

Reply:
How about leaving rtmros_tutorials for now, and fixing hrpsys_gazebo_atlas not to depend on rtmros_tutorials in anorther PR? (that PR can be postponed)
Or that solution cannot work on melodic?

Reply:
> You are preventing hrpsys_gazebo_atlas from being built, aren't you?

In this commit, `hrpsys_gazebo_atlas` is built, but not tested.
In this commit,
`BUILD_PKGS=""`,
`TEST_PKGS="eusgazebo hrp2jsk_moveit_config hrp2jsknt_moveit_config hrp2jsknts_moveit_config hrp2w_moveit_config hrpsys_gazebo_msgs hrpsys_gazebo_general samplerobot_moveit_config staro_moveit_config"`.

In `travis.sh`,
```
catkin build $BUILD_PKGS #build all packages
catkin build --catkin-make-args run_tests -- -i --no-deps --no-status $TEST_PKGS #not test hrpsys_gazebo_atlas
```

> fixing hrpsys_gazebo_atlas not to depend on rtmros_tutorials

It sounds good. I fixed this in https://github.com/start-jsk/rtmros_gazebo/pull/249 ( https://github.com/start-jsk/rtmros_gazebo/pull/249/commits/06a78e1e37fbd4fa716118a6143d68025cb4f88f )